.gitmodules

```diff
@@ -2,7 +2,7 @@
 	ignore = dirty
 	branch = master
 	path = vendor/miniupnp
-	url = https://github.com/miniupnp/miniupnp.git
+	url = https://github.com/status-im/miniupnp.git
```

Comment:
Why fork the repo?

Reply:
We have a local commit that's not merged upstream yet.